wgpu-core/src/lib.rs

```diff
@@ -17,6 +17,7 @@ pub mod backend {
     pub use gfx_backend_vulkan::Backend as Vulkan;
 }
 
+pub mod adapter;
```

Comment:
I moved some stuff into this module, but somebody who has more experience with `wgpu` should offer some feedback on what exactly should be moved here.